zuul.d/jobs.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- project:
+    check:
+      jobs:
+        - manila-operator-kuttl
+- job:
+    name: manila-operator-kuttl
+    parent: cifmw-kuttl
```

Comment:
we should go with `cifmw-multinode-kuttl` layout, which is more stable now, the previous one may not be working anymore.

Reply:
nice, testing that layout here: https://review.rdoproject.org/r/c/testproject/+/50787
The standalone one works and I'm wondering if it's enough to run kuttl tests, but +1 on moving to that layout as long as it's stable.